colorbleed/plugins/maya/publish/extract_yeti_rig.py

```diff
@@ -17,36 +17,40 @@ def disconnected_attributes(settings, members):
     try:
         for input in settings["inputs"]:
 
-            # get source
-            socket_id = input["socketID"]
-            sources = lib.lsattr("cbId", socket_id)
-            sources = [i for i in sources if
+            # Get source shapes
+            source_nodes = lib.lsattr("cbId", input["sourceID"])
+            sources = [i for i in source_nodes if
                        not cmds.referenceQuery(i, isNodeReferenced=True)
```

Comment:
This seems like odd logic for this context manager - why does it require the input to be a referenced node? If that's required than collection must already have required it and it must've been validated if it breaks otherwise.

Furthermore, I'm expecting to always find at most a single match - if there are more; what do we do? Is that an ambiguous error? Do we allow breaking both of them? Or do we want to assert there's always a single match so it's explicit about what connection is broken?